Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ssdc-rm-toolbox = {editable = true,git = "https://github.com/ONSdigital/ssdc-rm-toolbox", ref="7c99b6bf4bf3aca71bdeb2f5e05a6aa3040741d9"}
```

Comment:
```suggestion
ssdc-rm-toolbox = {editable = true,git = "git@github.com:ONSdigital/ssdc-rm-toolbox.git", ref="7c99b6bf4bf3aca71bdeb2f5e05a6aa3040741d9"}

```

Reply:
Will commit for now, might change back in future when repo public

Reply:
I think this might give us issues in build environments as it needs a key.